api/core/workflow/workflow_engine_manager.py

```diff
@@ -153,7 +156,7 @@ def run_workflow(self, workflow: Workflow,
                     raise ValueError('Max steps 50 reached.')
 
                 # or max execution time 10min reached
-                if self._is_timed_out(start_at=workflow_run_state.start_at, max_execution_time=600):
+                if self._is_timed_out(start_at=workflow_run_state.start_at, max_execution_time=WORKFLOW_MAX_EXECUTION_TIME):
                     raise ValueError('Max execution time 10min reached.')
```

Comment:
And recommended to keep the max_execution_time value in the error message consistent with the value of the environment variable.

Reply:
fix